python/tank/pipelineconfig_utils.py

```diff
@@ -183,148 +202,50 @@ def get_core_path_for_config(pipeline_config_path):
 
     In the case of a localized PC, it just returns the given path.
     Otherwise, it resolves the location via the core_xxxx.cfg files.
-    
+
     :param pipeline_config_path: path to a pipeline configuration
 
     :returns: Path to the studio location root or pipeline configuration root or None if not resolved
     """
     try:
-        return _get_core_path_for_config(pipeline_config_path)
-    except:
+        studio_folder = os.path.join(
```

Comment:
how does this work for localized vs shared cores? Some comments would be great. (I find the comments around the `..` a little hard to follow)